src/components/charts/line.js

```diff
@@ -474,6 +477,12 @@ export default class Line extends Component {
                         checked={pureRandom.animation}
                         onChange={this.onPureRandomCheckBoxChange}/>Enable Animation</label>
         </div>
+		<div className="checkbox">
+		  <label><input type="checkbox"
+						value={optionsPureRandom.area}
```

Comment:
Looks like here it still has quite irregular spacings, but here it is so much important... ;)